### DISCUSSION TITLE
How to setup multisite?

### DISCUSSION THREAD

**Discussion by git-gnulf** [givanz/Vvveb#222] [Q&A]
Hi all,

I want to test the multisite feature. I have found the settings in Settings > Sites and have already created a new entry for a new site. So far so good. But how do I configure nginx now? Do I create a new host.conf and simply point the root to the directory of the vvveb installation? Or is there something else to consider?

Do you have any tips?

**Comment by givanz** [comment]
Hi

You can use nginx `default_server` option https://nginx.org/en/docs/http/request_processing.html to catch all domains/subdomains that are not defined in other blocks.

```nginx
server {
    listen      80 default_server;
    server_name example.net www.example.net;
    ...
}
```

you can also use wildcards if you want to restrict it only for certain subdomains etc

```nginx
server {
    listen      80;
    server_name *.example.net *.anotherdomain.com;
    ...
}
```

For quick tests you can add your test domain/subdomain to `/etc/hosts` or `C:\Windows\System32\drivers\etc` on windows and point it to your server ip.
